api/enterprise-api-ref/two-factor-authentication.markdown

```diff
@@ -0,0 +1,239 @@
+---
+layout: default
+title: Two-factor authentication API
+published: true
+---
+The Two-factor authentication API enables users to add an extra layer of security to their accounts
+by requiring a TOTP (time-based one-time password) in addition to their primary credentials.
+
+Once two-factor authentication is enabled, requests to obtain an OAuth token, change a password, or
+disable two-factor authentication must include an additional header: Cf-2fa-Token: code.
```

Comment:
```suggestion
disable two-factor authentication must include an additional header: `Cf-2fa-Token: <code>`.
```

Maybe make it more clear that the head includes both the name: `Cf-2fa-Token` and a value?

Reply:
+1